### PR TITLE
quz

### PR DESCRIPTION
**Pull request by RyLuras**

### CODE REVIEW

**Comment by RyLuras** [comment]
ugh, i cracked under the time pressure. I just couldnt understand why i wasnt passing the first part for the longest time, I kept getting back a string, so thats where all my time got burned into. I realize now that I simply needed to JSON.stringify the result in my rest....

**Comment by mehtaphysical** [comment]
looks good. Missing a few tests and the delete route

File: lib/app.js [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+
+    } else if(req.method === 'GET' && path === '/api/penguins') {
+        res.setHeader('content-type', 'application/JSON');
+        let response = ['bernice', 'bernard'];
+        res.end(JSON.stringify(response));
+    }
+    
+    else if(req.method === 'GET' && path === '/api/penguins/king') {
+        if(url.query.format === 'full') {
+            res.end({
```
Comment (mehtaphysical) [inline lib/app.js]: This needs to be stringified

File: lib/app.js [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+    
+    else if(req.method === 'GET' && path === '/api/penguins/king') {
+        if(url.query.format === 'full') {
+            res.end({ 
+                name: 'bernice',
+                description: 'What a penguin!',
+                age: 7
+            });
+        }
+        else res.end({ name: 'bernice' });
```
Comment (mehtaphysical) [inline lib/app.js]: Stringify

**Pull request merged**